autodoc.pl

```diff
@@ -1398,21 +1398,43 @@ ($$$)
 
         print $fh "\nNOTE: the C<perl_$item_name()> form is B<deprecated>.\n"
                                                     if $item_flags =~ /O/;
-        # Is Perl_, but no #define foo # Perl_foo
-        if (   ($item_flags =~ /p/ && $item_flags =~ /o/ && $item_flags !~ /M/)
 
-                # Can't handle threaded varargs
-            || (   $item_flags =~ /f/
-                && $item_flags !~ /T/
-                && $item_name !~ /strftime/))
-        {
+        # If only the Perl_foo form is to be displayed, change the name of
+        # this item to be that.  This happens for either of two reasons:
+        #   1) The flags say we want "Perl_", but also to not create an entry
```

Comment:
`but also to not create an entry ...` -- confused as to what that's supposed to mean.  Did you intend `do not create`?  Or perhaps, `but we also do not want to create an entry ...`?

Reply:
Normally we define functions in the API with a `Perl_` prefix, *and* define a macro without the prefix to call that function, adding a thread context parameter if needed, eg.
```
/* proto.h */
PERL_CALLCONV void
Perl_sv_setpvn(pTHX_ SV * const sv, const char * const ptr, const STRLEN len);

/* embed.h */
# define sv_setpvn(a,b,c)                       Perl_sv_setpvn(aTHX_ a,b,c)
```
In some cases we have the `Perl_` prefix but don't want the macro to be defined.

Note that the code being modified here doesn't define the macro, it's just documenting how to call the function, so "to not create" is more reasonable than "do not create".

Reply:
Unchanged